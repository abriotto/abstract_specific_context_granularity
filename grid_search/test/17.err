  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  7%|#############2                                                                                                                                                                              | 7/99 [00:00<00:01, 55.90it/s] 14%|##########################4                                                                                                                                                                | 14/99 [00:00<00:01, 60.24it/s] 23%|###########################################4                                                                                                                                               | 23/99 [00:00<00:01, 65.54it/s] 30%|########################################################6                                                                                                                                  | 30/99 [00:00<00:01, 64.73it/s] 37%|#####################################################################8                                                                                                                     | 37/99 [00:00<00:01, 60.44it/s] 44%|###################################################################################1                                                                                                       | 44/99 [00:00<00:00, 61.97it/s] 57%|#########################################################################################################7                                                                                 | 56/99 [00:00<00:00, 73.51it/s] 65%|########################################################################################################################8                                                                  | 64/99 [00:00<00:00, 70.17it/s] 73%|########################################################################################################################################                                                   | 72/99 [00:01<00:00, 63.09it/s] 80%|#####################################################################################################################################################2                                     | 79/99 [00:01<00:00, 63.22it/s] 87%|##################################################################################################################################################################4                        | 86/99 [00:01<00:00, 56.51it/s] 94%|###############################################################################################################################################################################6           | 93/99 [00:01<00:00, 58.37it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 57.33it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 61.65it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 20%|#####################################6                                                                                                                                                      | 5/25 [00:00<00:00, 45.48it/s] 44%|##################################################################################2                                                                                                        | 11/25 [00:00<00:00, 50.85it/s] 68%|###############################################################################################################################1                                                           | 17/25 [00:00<00:00, 47.66it/s] 88%|####################################################################################################################################################################5                      | 22/25 [00:00<00:00, 48.25it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 48.22it/s]
