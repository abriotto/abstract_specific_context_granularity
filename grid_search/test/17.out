# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=32", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6948316097259521, "acc": 0.5142518877983093, "length": 3.5911059379577637, "mode": "train", "epoch": 1}
{"loss": 0.687755823135376, "acc": 0.5523810386657715, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6944704055786133, "acc": 0.5248159766197205, "length": 3.3022620677948, "mode": "train", "epoch": 2}
{"loss": 0.6880823373794556, "acc": 0.5466667413711548, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.678914487361908, "acc": 0.5809236168861389, "length": 3.024196147918701, "mode": "train", "epoch": 3}
{"loss": 0.6379435062408447, "acc": 0.6709385514259338, "length": 3.537142753601074, "mode": "test", "epoch": 3}
{"loss": 0.6124147772789001, "acc": 0.6703994274139404, "length": 3.610142946243286, "mode": "train", "epoch": 4}
{"loss": 0.594459593296051, "acc": 0.7123809456825256, "length": 4.0, "mode": "test", "epoch": 4}
{"loss": 0.5625704526901245, "acc": 0.7033841013908386, "length": 3.8678359985351562, "mode": "train", "epoch": 5}
{"loss": 0.5541577935218811, "acc": 0.725714385509491, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.5192167162895203, "acc": 0.7437768578529358, "length": 3.9021337032318115, "mode": "train", "epoch": 6}
{"loss": 0.5472031235694885, "acc": 0.7447620630264282, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.48357218503952026, "acc": 0.7628523111343384, "length": 3.9336044788360596, "mode": "train", "epoch": 7}
{"loss": 0.5418761968612671, "acc": 0.7523809671401978, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.4722658693790436, "acc": 0.7703067064285278, "length": 3.9526381492614746, "mode": "train", "epoch": 8}
{"loss": 0.5305920243263245, "acc": 0.7647619843482971, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.4605291187763214, "acc": 0.7933551073074341, "length": 3.961353063583374, "mode": "train", "epoch": 9}
{"loss": 0.5359588861465454, "acc": 0.7780952453613281, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.4494328498840332, "acc": 0.7996762990951538, "length": 3.968327045440674, "mode": "train", "epoch": 10}
{"loss": 0.5188480615615845, "acc": 0.7761905193328857, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.4330686032772064, "acc": 0.8126932382583618, "length": 3.9757416248321533, "mode": "train", "epoch": 11}
{"loss": 0.5214064121246338, "acc": 0.7685714364051819, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.4278092384338379, "acc": 0.8165786266326904, "length": 3.980476140975952, "mode": "train", "epoch": 12}
{"loss": 0.5354006290435791, "acc": 0.7657143473625183, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.41844409704208374, "acc": 0.827133059501648, "length": 3.9821908473968506, "mode": "train", "epoch": 13}
{"loss": 0.5204609632492065, "acc": 0.7742857933044434, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.40399664640426636, "acc": 0.8220577836036682, "length": 3.9851157665252686, "mode": "train", "epoch": 14}
{"loss": 0.5073966383934021, "acc": 0.7800000905990601, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.3958238661289215, "acc": 0.8331625461578369, "length": 3.9874300956726074, "mode": "train", "epoch": 15}
{"loss": 0.5102421641349792, "acc": 0.7809523344039917, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.38804879784584045, "acc": 0.8347727060317993, "length": 3.9904794692993164, "mode": "train", "epoch": 16}
{"loss": 0.5176622867584229, "acc": 0.7904763221740723, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.3806077539920807, "acc": 0.8426598906517029, "length": 3.989530324935913, "mode": "train", "epoch": 17}
{"loss": 0.5198947191238403, "acc": 0.7857143878936768, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.37559249997138977, "acc": 0.8433544039726257, "length": 3.992546796798706, "mode": "train", "epoch": 18}
{"loss": 0.5306764841079712, "acc": 0.7895238399505615, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.3658720850944519, "acc": 0.8500831723213196, "length": 3.993776798248291, "mode": "train", "epoch": 19}
{"loss": 0.5331512689590454, "acc": 0.7771428823471069, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.35654377937316895, "acc": 0.8551674485206604, "length": 3.994277238845825, "mode": "train", "epoch": 20}
{"loss": 0.5106490850448608, "acc": 0.7933334112167358, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.3482515513896942, "acc": 0.8612069487571716, "length": 3.994499921798706, "mode": "train", "epoch": 21}
{"loss": 0.5148698091506958, "acc": 0.771428644657135, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.3399694263935089, "acc": 0.8580685257911682, "length": 3.995743751525879, "mode": "train", "epoch": 22}
{"loss": 0.5207309722900391, "acc": 0.7809523344039917, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.332861065864563, "acc": 0.8666399121284485, "length": 3.996220827102661, "mode": "train", "epoch": 23}
{"loss": 0.5129613280296326, "acc": 0.7914285659790039, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.33687955141067505, "acc": 0.8555369973182678, "length": 3.9965415000915527, "mode": "train", "epoch": 24}
{"loss": 0.49651122093200684, "acc": 0.795238196849823, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.33003759384155273, "acc": 0.8647561073303223, "length": 3.996872663497925, "mode": "train", "epoch": 25}
{"loss": 0.5271828174591064, "acc": 0.7942857146263123, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.3192572295665741, "acc": 0.8644310235977173, "length": 3.9965195655822754, "mode": "train", "epoch": 26}
{"loss": 0.48992055654525757, "acc": 0.803809642791748, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.3093811273574829, "acc": 0.8711259961128235, "length": 3.9972212314605713, "mode": "train", "epoch": 27}
{"loss": 0.47960135340690613, "acc": 0.8190475702285767, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.3053553104400635, "acc": 0.8727166652679443, "length": 3.9974327087402344, "mode": "train", "epoch": 28}
{"loss": 0.509262204170227, "acc": 0.7904762625694275, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.2935112714767456, "acc": 0.8784371614456177, "length": 3.997152805328369, "mode": "train", "epoch": 29}
{"loss": 0.4768643379211426, "acc": 0.8142858147621155, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.2887762188911438, "acc": 0.8857540488243103, "length": 3.9975180625915527, "mode": "train", "epoch": 30}
{"loss": 0.49394476413726807, "acc": 0.7876189947128296, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.28334715962409973, "acc": 0.8838505744934082, "length": 3.9976508617401123, "mode": "train", "epoch": 31}
{"loss": 0.5072423219680786, "acc": 0.8000000715255737, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.2752287685871124, "acc": 0.8851211667060852, "length": 3.997742176055908, "mode": "train", "epoch": 32}
{"loss": 0.4746339023113251, "acc": 0.8142856955528259, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.27585187554359436, "acc": 0.8886246085166931, "length": 3.9980859756469727, "mode": "train", "epoch": 33}
{"loss": 0.45824679732322693, "acc": 0.8104761838912964, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.2763683795928955, "acc": 0.8803655505180359, "length": 3.998102903366089, "mode": "train", "epoch": 34}
{"loss": 0.4525342881679535, "acc": 0.8219048380851746, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.2639092206954956, "acc": 0.889553964138031, "length": 3.9968066215515137, "mode": "train", "epoch": 35}
{"loss": 0.46173661947250366, "acc": 0.8104761838912964, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.25385329127311707, "acc": 0.8952978253364563, "length": 3.9979238510131836, "mode": "train", "epoch": 36}
{"loss": 0.4913176894187927, "acc": 0.8028572201728821, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.25793224573135376, "acc": 0.8965635895729065, "length": 3.998133897781372, "mode": "train", "epoch": 37}
{"loss": 0.46942126750946045, "acc": 0.8171430230140686, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.25336337089538574, "acc": 0.8988118767738342, "length": 3.998530864715576, "mode": "train", "epoch": 38}
{"loss": 0.5046701431274414, "acc": 0.7904763221740723, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.24138593673706055, "acc": 0.9058047533035278, "length": 3.998530387878418, "mode": "train", "epoch": 39}
{"loss": 0.4598779082298279, "acc": 0.8266667127609253, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.2404542863368988, "acc": 0.9016742706298828, "length": 3.998560667037964, "mode": "train", "epoch": 40}
{"loss": 0.44842469692230225, "acc": 0.8257143497467041, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.22659938037395477, "acc": 0.9137580394744873, "length": 3.998603105545044, "mode": "train", "epoch": 41}
{"loss": 0.42329663038253784, "acc": 0.8219048976898193, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.23332874476909637, "acc": 0.9092906713485718, "length": 3.9982783794403076, "mode": "train", "epoch": 42}
{"loss": 0.49692270159721375, "acc": 0.8219048976898193, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.2425013929605484, "acc": 0.9086730480194092, "length": 3.9987540245056152, "mode": "train", "epoch": 43}
{"loss": 0.4793417155742645, "acc": 0.832381010055542, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.22653593122959137, "acc": 0.9080188274383545, "length": 3.9982099533081055, "mode": "train", "epoch": 44}
{"loss": 0.4175044000148773, "acc": 0.831428587436676, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.23116886615753174, "acc": 0.9077009558677673, "length": 3.998047351837158, "mode": "train", "epoch": 45}
{"loss": 0.4660191237926483, "acc": 0.8276191353797913, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.2193019539117813, "acc": 0.9095981121063232, "length": 3.9979662895202637, "mode": "train", "epoch": 46}
{"loss": 0.4657894968986511, "acc": 0.8142856955528259, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.21965323388576508, "acc": 0.9168429970741272, "length": 3.996318817138672, "mode": "train", "epoch": 47}
{"loss": 0.4445027709007263, "acc": 0.8266665935516357, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.20557306706905365, "acc": 0.9194980263710022, "length": 3.999037027359009, "mode": "train", "epoch": 48}
{"loss": 0.43979665637016296, "acc": 0.8257143497467041, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.20442718267440796, "acc": 0.9197936654090881, "length": 3.9986016750335693, "mode": "train", "epoch": 49}
{"loss": 0.4395918846130371, "acc": 0.8342857956886292, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.19384115934371948, "acc": 0.9290028810501099, "length": 3.998380422592163, "mode": "train", "epoch": 50}
{"loss": 0.44968003034591675, "acc": 0.8276190161705017, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.18964658677577972, "acc": 0.9296431541442871, "length": 3.998556137084961, "mode": "train", "epoch": 51}
{"loss": 0.4456990957260132, "acc": 0.8380952477455139, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.17762410640716553, "acc": 0.9353876709938049, "length": 3.998958110809326, "mode": "train", "epoch": 52}
{"loss": 0.4543904662132263, "acc": 0.831428587436676, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.20403680205345154, "acc": 0.924896776676178, "length": 3.999023914337158, "mode": "train", "epoch": 53}
{"loss": 0.4674240052700043, "acc": 0.8390477895736694, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.20529557764530182, "acc": 0.9252052903175354, "length": 3.998654365539551, "mode": "train", "epoch": 54}
{"loss": 0.4552184045314789, "acc": 0.8371428847312927, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.18789686262607574, "acc": 0.9261531829833984, "length": 3.998626470565796, "mode": "train", "epoch": 55}
{"loss": 0.4480314254760742, "acc": 0.8399999141693115, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.1874481588602066, "acc": 0.9322165250778198, "length": 3.999154806137085, "mode": "train", "epoch": 56}
{"loss": 0.4376436173915863, "acc": 0.8571427464485168, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.1703041046857834, "acc": 0.9331560134887695, "length": 3.9988067150115967, "mode": "train", "epoch": 57}
{"loss": 0.45202207565307617, "acc": 0.8428571224212646, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.16988825798034668, "acc": 0.9356383085250854, "length": 3.9974629878997803, "mode": "train", "epoch": 58}
{"loss": 0.4468567371368408, "acc": 0.844761848449707, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.1653813272714615, "acc": 0.9366704821586609, "length": 3.999133348464966, "mode": "train", "epoch": 59}
{"loss": 0.49940332770347595, "acc": 0.8304763436317444, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.1791505217552185, "acc": 0.9264947175979614, "length": 3.9990928173065186, "mode": "train", "epoch": 60}
{"loss": 0.4875681400299072, "acc": 0.8304761648178101, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.16793888807296753, "acc": 0.9350770115852356, "length": 3.998975992202759, "mode": "train", "epoch": 61}
{"loss": 0.46108895540237427, "acc": 0.8419047594070435, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.15860213339328766, "acc": 0.9420558214187622, "length": 3.998581647872925, "mode": "train", "epoch": 62}
{"loss": 0.48241424560546875, "acc": 0.8400000929832458, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.1595965474843979, "acc": 0.9370043873786926, "length": 3.999544382095337, "mode": "train", "epoch": 63}
{"loss": 0.45457351207733154, "acc": 0.8457143902778625, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.1533736139535904, "acc": 0.9474887251853943, "length": 3.999387264251709, "mode": "train", "epoch": 64}
{"loss": 0.4683871865272522, "acc": 0.841904878616333, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.1596783548593521, "acc": 0.9408149719238281, "length": 3.9993896484375, "mode": "train", "epoch": 65}
{"loss": 0.4996139705181122, "acc": 0.8342857956886292, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.15167154371738434, "acc": 0.9439966678619385, "length": 3.9994218349456787, "mode": "train", "epoch": 66}
{"loss": 0.491013765335083, "acc": 0.8409525156021118, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.1489303857088089, "acc": 0.9481353759765625, "length": 3.9995980262756348, "mode": "train", "epoch": 67}
{"loss": 0.4707052409648895, "acc": 0.8514285683631897, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.13914479315280914, "acc": 0.9516290426254272, "length": 3.9995644092559814, "mode": "train", "epoch": 68}
{"loss": 0.4412742257118225, "acc": 0.8590476512908936, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.1387098878622055, "acc": 0.9516403675079346, "length": 3.9996893405914307, "mode": "train", "epoch": 69}
{"loss": 0.4920494556427002, "acc": 0.8380951881408691, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.13758714497089386, "acc": 0.9503570795059204, "length": 3.999605894088745, "mode": "train", "epoch": 70}
{"loss": 0.44247812032699585, "acc": 0.8523808717727661, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.142574280500412, "acc": 0.9484564661979675, "length": 3.9996728897094727, "mode": "train", "epoch": 71}
{"loss": 0.5101341605186462, "acc": 0.8419048190116882, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.13599836826324463, "acc": 0.9510130286216736, "length": 3.999884843826294, "mode": "train", "epoch": 72}
{"loss": 0.48117363452911377, "acc": 0.8523808717727661, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.13381831347942352, "acc": 0.9491050243377686, "length": 3.9998714923858643, "mode": "train", "epoch": 73}
{"loss": 0.4634370803833008, "acc": 0.854285717010498, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.13061584532260895, "acc": 0.9538572430610657, "length": 3.999567985534668, "mode": "train", "epoch": 74}
{"loss": 0.5061590671539307, "acc": 0.8447620272636414, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.13439030945301056, "acc": 0.950692892074585, "length": 3.9998443126678467, "mode": "train", "epoch": 75}
{"loss": 0.48488348722457886, "acc": 0.8552380204200745, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.12179642915725708, "acc": 0.9586386680603027, "length": 3.999741792678833, "mode": "train", "epoch": 76}
{"loss": 0.5044260025024414, "acc": 0.8504762649536133, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.11679819226264954, "acc": 0.9599146842956543, "length": 3.999826192855835, "mode": "train", "epoch": 77}
{"loss": 0.48874175548553467, "acc": 0.8600000739097595, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.12696687877178192, "acc": 0.953550398349762, "length": 3.999757766723633, "mode": "train", "epoch": 78}
{"loss": 0.5040712356567383, "acc": 0.854285717010498, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.12577931582927704, "acc": 0.9564189314842224, "length": 3.9998676776885986, "mode": "train", "epoch": 79}
{"loss": 0.4791663587093353, "acc": 0.8457143902778625, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.1349814534187317, "acc": 0.9468727111816406, "length": 3.9997236728668213, "mode": "train", "epoch": 80}
{"loss": 0.4816931188106537, "acc": 0.8542855978012085, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.12128014117479324, "acc": 0.9592829346656799, "length": 3.9999115467071533, "mode": "train", "epoch": 81}
{"loss": 0.4879000782966614, "acc": 0.8628571629524231, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.13588230311870575, "acc": 0.9500541687011719, "length": 3.999800682067871, "mode": "train", "epoch": 82}
{"loss": 0.48371395468711853, "acc": 0.8580951690673828, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.12721361219882965, "acc": 0.9522579312324524, "length": 3.9994096755981445, "mode": "train", "epoch": 83}
{"loss": 0.4550974369049072, "acc": 0.8609524965286255, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.13048575818538666, "acc": 0.9538766145706177, "length": 3.999891996383667, "mode": "train", "epoch": 84}
{"loss": 0.5140084028244019, "acc": 0.8523810505867004, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.1235140934586525, "acc": 0.9557794332504272, "length": 3.9997119903564453, "mode": "train", "epoch": 85}
{"loss": 0.47479915618896484, "acc": 0.8571429252624512, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.12466837465763092, "acc": 0.9554675817489624, "length": 3.999924898147583, "mode": "train", "epoch": 86}
{"loss": 0.4619565010070801, "acc": 0.8657143712043762, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.11524663120508194, "acc": 0.9573728442192078, "length": 3.9998629093170166, "mode": "train", "epoch": 87}
{"loss": 0.45828112959861755, "acc": 0.8580951690673828, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.11358848214149475, "acc": 0.9618040323257446, "length": 3.999575614929199, "mode": "train", "epoch": 88}
{"loss": 0.5000337958335876, "acc": 0.8447619676589966, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.11274299770593643, "acc": 0.9653244018554688, "length": 3.999882698059082, "mode": "train", "epoch": 89}
{"loss": 0.4729665517807007, "acc": 0.85619056224823, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.11575397104024887, "acc": 0.960212767124176, "length": 3.999234914779663, "mode": "train", "epoch": 90}
{"loss": 0.46118879318237305, "acc": 0.858095109462738, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.1125325933098793, "acc": 0.958648681640625, "length": 3.999943971633911, "mode": "train", "epoch": 91}
{"loss": 0.4733757972717285, "acc": 0.8571428656578064, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.11955693364143372, "acc": 0.9567395448684692, "length": 3.999922752380371, "mode": "train", "epoch": 92}
{"loss": 0.47182536125183105, "acc": 0.8657143712043762, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.10846707969903946, "acc": 0.9599201679229736, "length": 3.999929666519165, "mode": "train", "epoch": 93}
{"loss": 0.4594686031341553, "acc": 0.8619047999382019, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.10580786317586899, "acc": 0.9624645709991455, "length": 3.9999239444732666, "mode": "train", "epoch": 94}
{"loss": 0.46476396918296814, "acc": 0.870476245880127, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.10128111392259598, "acc": 0.9669188261032104, "length": 3.999955177307129, "mode": "train", "epoch": 95}
{"loss": 0.41671091318130493, "acc": 0.8695237636566162, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.09541904181241989, "acc": 0.9694604873657227, "length": 3.999906301498413, "mode": "train", "epoch": 96}
{"loss": 0.44172680377960205, "acc": 0.8714284896850586, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.09653995931148529, "acc": 0.9631026387214661, "length": 3.999969244003296, "mode": "train", "epoch": 97}
{"loss": 0.43447285890579224, "acc": 0.8733334541320801, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.09875447303056717, "acc": 0.9649832248687744, "length": 3.9994959831237793, "mode": "train", "epoch": 98}
{"loss": 0.4811815023422241, "acc": 0.8600000739097595, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.10199816524982452, "acc": 0.9637395143508911, "length": 3.999972105026245, "mode": "train", "epoch": 99}
{"loss": 0.4878501296043396, "acc": 0.8638096451759338, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.1121177226305008, "acc": 0.9586504101753235, "length": 3.9999804496765137, "mode": "train", "epoch": 100}
{"loss": 0.4938145875930786, "acc": 0.8619047403335571, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.7752902507781982
